spdm_dump/spdm/spdm_dump_session.c

```diff
@@ -20,6 +20,8 @@ void *m_dhe_secret_buffer;
 size_t m_dhe_secret_buffer_size;
 void *m_psk_buffer;
 size_t m_psk_buffer_size;
+uint8_t m_use_slot_id = 0;
+uint8_t m_use_slot_count = 3;
```

Comment:
why 3 ?

Reply:
just refer to m_use_slot_count in spdm_emu.

Reply:
That is not right.

spdm_dump should be independent with spdm_emu.